lib/options.js

```diff
@@ -191,6 +191,12 @@ module.exports = optionator({
             description: "Automatically fix problems"
         },
         {
+            option: "fix-dry-run",
+            type: "Boolean",
+            default: false,
+            description: "Automatically fix problems without saving the changes to the file system"
+        },
+        {
```

Comment:
Can you update the [CLI docs](https://github.com/eslint/eslint/blob/ffa021e7696b24722cbbef306d494dd64d7b5215/docs/user-guide/command-line-interface.md) with information on the new flag? There are two places to update:

1. The options summary [here](https://github.com/eslint/eslint/blob/ffa021e7696b24722cbbef306d494dd64d7b5215/docs/user-guide/command-line-interface.md#options) should be updated (I think you can just run `bin/eslint.js -h` on your branch and copy-paste the result).
1. A section should be added [here](https://github.com/eslint/eslint/blob/ffa021e7696b24722cbbef306d494dd64d7b5215/docs/user-guide/command-line-interface.md#--fix). This could describe the flag in a bit more detail, and possibly explain its use cases. For example, it might be worth mentioning that the flag should be used with a formatter that includes the output (such as `json`) if the user/integration wants to access the autofix results. It could also be a good idea to modify the description of `--fix` in the section above to recommend using `--fix-dry-run` for piped-in code.